rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -493,6 +519,23 @@ def _decode_transaction(
             counterparties=counterparties,
         )
 
+        if monerium_special_handling_event is True:
+            # When events that need special handling exist iterate over the decoded events an
+            # exchange the legacy assets by the v2 assets. Also delete v2 events to
+            # avoid duplications. V2 events exist only for the case of
+            # interacting with the v1 tokens since interacting with v2 doesn't emit v1 transfers.
```

Comment:
```suggestion
            # avoid duplications. In the case of this exception handling, v2 events exist only for the case of
            # interacting with the v1 tokens since interacting with v2 doesn't emit v1 transfers.
```